src/react-intl.js

```diff
@@ -14,6 +14,7 @@ export {intlShape} from './types';
 export {default as defineMessages} from './define-messages';
 
 export {default as injectIntl, Context as IntlContext} from './components/withIntl';
+export {default as useIntl} from './components/useIntl';
```

Comment:
Please add a check towards wether or not the React version is gte. `16.8.0` (supports hooks), eg.:

```js
const [ major, minor ] = React.version.split('.')
if (major >= 17 || minor >= 8) {
  // export
}
```

Reply:
I was thinking this too, but does it really required?! The user won't be able to execute hook anyway (since `useContext` is not a function). So I think it's kind of redundant... but I would be happy to add this if there are other concerns (e.g. it allows tree shaking to work or something else?)

Reply:
You're right, it shouldn't make any difference after all :rofl::+1: